torch/_inductor/compile_fx.py

```diff
@@ -336,7 +352,9 @@ def count_bytes_inner(
         _recursive_post_grad_passes(gm, False)
 
     graph = GraphLowering(gm, shape_env=shape_env, num_static_inputs=num_fixed)
-    with V.set_graph_handler(graph), V.set_real_inputs(example_inputs):
+    with V.set_graph_handler(graph), V.set_real_inputs(
+        example_inputs
+    ), maybe_disable_comprehensive_padding(example_inputs):
```

Comment:
any reason to do this as a config patch instead of as a field on GraphLowering?

Reply:
Nope, either way should work. I can access the config by either config.comprehensive_padding or V.graph.comprehensive_padding .